src/components/Card.js

```diff
@@ -20,12 +30,36 @@ export const Card = ({ product }) => {
         <p className="h3 colored-text">{price} ₹</p>
       </div>
       <div className="card-actions p-2 row-flex full-wd">
-        <div className="card-btns">
-          <button className="btn primary-btn m-r-1">
-            <i className="fa fa-shopping-cart" aria-hidden="true"></i>
-            <span className="p-l-1">Add to cart</span>
-          </button>
-        </div>
+        {!cartItem ? (
+          <div className="card-btns">
+            <button
+              className="btn primary-btn m-r-1"
+              onClick={() => dispatchCart({ type: "ADDTOCART", payload: product })}
+            >
+              <i className="fa fa-shopping-cart" aria-hidden="true"></i>
+              <span className="p-l-1">Add to cart</span>
+            </button>
+          </div>
+        ) : (
+          <div className="quantity-selector row-flex no-wrap full-height">
+            <i
+              className="fa fa-trash-o light-font m-r-2 red-text"
+              aria-hidden="true"
+              onClick={() => dispatchCart({ type: "DELETEFROMCART", payload: product })}
+            ></i>
+            <i
+              className="fa fa-minus light-font m-r-2"
+              aria-hidden="true"
+              onClick={() => dispatchCart({ type: "DECREMENTFROMCART", payload: product })}
+            ></i>
+            <p className="quantity m-r-2 h3">{cartItem?.count}</p>
+            <i
+              className="fa fa-plus light-font"
+              aria-hidden="true"
+              onClick={() => dispatchCart({ type: "ADDTOCART", payload: product })}
```

Comment:
Just a nitpick you can use snake case in the type value. It will be more readable. E.g ```dispatchCart({type:"DECREMENT_FROM_CART",payload:product})```

Reply:
Yes, thanks for the suggestion. I've made the said changes.